src/main/java/account/Account.java

```diff
@@ -0,0 +1,45 @@
+package account;
```

Comment:
Can we move this file under the app/java/com/courseproject/tindar/entities? And delete src/main/java/account folder?